shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -615,5 +620,21 @@ private float min(float a, float b, float c, float d) {
         private float max(float a, float b, float c, float d) {
             return Math.max(a, Math.max(b, Math.max(c, d)));
         }
+
+        private String getValueLabelHint() {
+            StringBuilder sb = new StringBuilder();
+            boolean first = true;
+            String[] array = { value, label, hint };
+            for (String word: array) {
+                if (word != null && (word = word.trim()).length() > 0) {
+                    if (first)
```

Comment:
this can be changed to:
  if (sb.length() > 0) sb.append(', ');

Reply:
I would imagine `if (!sb.isEmpty())` is better than `if (sb.length() > 0)` still.

Reply:
Looks like StringBuilder doesn't have an isEmpty method: https://docs.oracle.com/javase/7/docs/api/java/lang/StringBuilder.html

Reply:
Ah, got you.